locale/en/blog/weekly-updates/weekly-update.2017-02-10.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Weekly Update - February 10th, 2017
+author: Minwoo Jung (github.com/JungMinu)
+date: 2017-02-10T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2017-02-10
+layout: blog-post.hbs
+---
+
+### Node v7.5.0 (Current)
+
+* [Node v7.5.0 (Current)](https://nodejs.org/en/blog/release/v7.5.0/). The complete changelog can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### Node v4.7.3 (LTS)
+
+* [Node v4.7.3 (LTS)](https://nodejs.org/en/blog/release/v4.7.3/). The complete changelog can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### Node v6.9.5 (LTS)
+
+* [Node v6.9.5 (LTS)](https://nodejs.org/en/blog/release/v6.9.5/). The complete changelog can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```

Comment:
Why don't we provide the direct link? https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V6.md#6.9.5